elastica/rod/knot_theory.py

```diff
@@ -130,13 +132,16 @@ def compute_link(
         type_of_additional_segment : str
             Determines the method to compute new segments (elements) added to the rod.
             Valid inputs are "next_tangent", "end_to_end", "net_tangent", otherwise program uses the center line.
+        alpha : float
+            Empirical factor to scale the segment length, where segment length is the rest length of the rod.
```

Comment:
Maybe add a little description on how to use?

Reply:
I added. @skim0119 can you check and approve.